chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/StatisticsController.java

```diff
@@ -51,6 +52,11 @@ public NonconformityTypeStatisticsResult getNonconformityCountByCriteria() {
         return response;
     }
 
+
+    @Deprecated
```

Comment:
I sort of thought we might stop calculating these statistics as well. Since the job wasn't changed when is the plan for stopping calculating and saving this data?